pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -104,14 +104,14 @@ public class DataProperties {
 
   private String fhirServerOAuthClientSecret;
 
-  private String structureDefinitionsDir;
-
-  private String structureDefinitionsClasspath;
+  private String structureDefinitionsPath;
 
   private int rowGroupSizeForParquetFiles;
 
   private FhirVersionEnum fhirVersion;
 
+  private int recursiveDepth;
```

Comment:
Can we add a validation to check if this value is a positive number, alternatively ignore if this is <= 0 (corresponding comments to be added in `application.yml` file)